calico_node/build.sh

```diff
@@ -9,11 +9,17 @@ echo "http://alpine.gliderlabs.com/alpine/edge/testing" >> /etc/apk/repositories
 apk -U add runit python py-setuptools libffi ip6tables ipset iputils iproute2
 
 # These packages are only used for building and get removed.
-apk add --virtual temp python-dev libffi-dev py-pip alpine-sdk curl
+apk add --virtual temp python-dev libffi-dev py-pip alpine-sdk curl cmake
 
 # Install Confd
 curl -L https://github.com/projectcalico/confd/releases/download/v0.10.0-scale/confd.static -o /sbin/confd
 
+# Install Felix dependency on yajl2
+curl -L https://github.com/lloyd/yajl/archive/2.1.0.tar.gz -o /tmp/2.1.0.tar.gz
+tar zxf /tmp/2.1.0.tar.gz
+cd ./yajl-2.1.0 && ./configure && make install && cd .. && rm -r ./yajl-2.1.0
+
+
```

Comment:
Perhaps I've missed something, but why build yajl 2.1.0 from source instead of just installing the package?

https://pkgs.alpinelinux.org/package/main/x86_64/yajl